CPPLINT.cfg

```diff
@@ -0,0 +1,2 @@
+filter=-build/include_subdir
+linelength=80
```

Comment:
ColumnLimit in clang-format set to 100, either change this to 100 or ColumnLimit to 80